modules/api/src/main/Env.scala

```diff
@@ -2,26 +2,22 @@ package lila.api
 
 import akka.actor.*
 import com.softwaremill.macwire.*
-import play.api.{ Configuration, Mode }
+import play.api.Mode
 
 import lila.chat.{ GetLinkCheck, IsChatFresh }
 import lila.common.Bus
 import lila.core.misc.lpv.*
 
 @Module
 final class Env(
-    appConfig: Configuration,
```

Comment:
There is probably some changes related to `Configuration`, I removed a lot of them in this PR.